extensions/amp-list/0.1/amp-list.js

```diff
@@ -130,6 +131,19 @@ export class AmpList extends AMP.BaseElement {
      */
     this.layoutCompleted_ = false;
 
+    /**
+     * Whether the amp-list has an initial `layout` value of `container`.
+     * If so, allow it to manage its own resizing in accordance to
+     * (1) requiring a placeholder to detect initial sizing and
+     * (2) fixing height on content change to prevent content jumping
+     *
+     * TODO: This configuration should eventually allow resizing to
```

Comment:
How would the mechanism change here? It would be useful to add an issue describing it.